### ISSUE TITLE
Move assemblies back to Galaxy

### ISSUE THREAD

**Issue by madisonkeene** [gvlproject/gvl.ansible.smrt_analysis#8]
How to get assemblies (.fasta files) from SMRT portal back to Galaxy to view (e.g. in JBrowse) without going via a local computer? (Or go via GenomeSpace?) - Data libraries via the API. Madi knows how to do it.

**Comment by madisonkeene** [comment]
@AnnaSyme @Slugger70 do either of you happen to know where these .fasta files end up living (like a directory)? Or have a server that I can check out to see where they are :) Then I can whip up a script to do this

**Comment by Slugger70** [comment]
There's a link to the file in the smrt portal interface.. http://abrpi.genome.edu.au/smrtportal/api/jobs/16449/contents/data/polished_assembly.fasta.gz is the link which would translate to: /mnt/gvl/apps/smrtanalysis/userdata/jobs/016/016449/data/polished_assembly.fasta.gz on the actual machine.
So we need a job number as an input then search the appropriate location for a polished_assembly.fasta.gz file and rename it in the data library

**Comment by madisonkeene** [comment]
Hmmmm OK... do we know where the `016` part of that path comes from?

**Comment by Slugger70** [comment]
Yes. Its the first 3 digits of the job number.. It's so there aren't too many directories in the jobs folder.

**Comment by madisonkeene** [comment]
If I generalize the data library script you'll be able to use it in this situation @Slugger70 - two birds one stone.

**Comment by Slugger70** [comment]
Yes indeed! That was one of the reasons I was thinking about it..
